src/opera/commands/deploy.py

```diff
@@ -28,6 +28,14 @@ def add_parser(subparsers):
             threads (positive number, default 1)",
         type=int, default=1
     )
+    parser.add_argument(
+        "--resume", "-r", action='store_true',
+        help="Resume the deployment from where it was interrupted.",
+    )
+    parser.add_argument(
+        "--force", "-f", action='store_true',
+        help="Delete the previous instance model and start over the deployment.",
```

Comment:
Here with `--force` option we are deleting the instance model that gets created with `opera deploy` and is present in `.opera/instances` folder. With these files gone the deployment can be started again without un-deploying anything. So if we want to go this way `opera redeploy` could be introduced as the new command that would undeploy the instances first and then it would deploy them again.

Reply:
No need for a new "redeploy" action. Even though the message is quite clear, I would vote for changing it to

"Clear orchestrator metadata explaining the state of the deployment (instance model) and start over the deployment process. Already deployed instances will not be undeployed."

Reply:
I actually like @anzoman 's original message better. At best, if you feel "instance model" is not something the users would understand (even though it is a term in the standard), you can go for "Delete the previous deployment state ..." or "Abandon deployment state and start over ...".

Reply:
I would prefer to emphasise "state". Just because the initial sentence is too similar to the "Delete the previous instance and start over...".

Reply:
Ok, I've refactored the message to `Delete the previous deployment state and start over the deployment.`